virl/files/mitaka.vinstall.py

```diff
@@ -453,7 +453,7 @@ def building_salt_all():
 keystone.password: {ospassword}
 keystone.tenant: admin
 keystone.tenant_id: {tenid}
-keystone.auth_url: {auth_url}/
+keystone.auth_url: 'http://127.0.0.1:5000/v2.0/'
```

Comment:
they are being installed because I fixed it on wednesday, look into mitaka.d/openstack.conf - this is a known problem and verified solution